.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,5 +16,3 @@ Choose one
 - [ ] Confirm DB Migration upgrade and downgrade tested.
 - [ ] Introduces new feature or API
 - [ ] Removes existing feature or API
-
-### REVIEWERS
```

Comment:
@mistercrunch why the removal of the reviewers section?

Reply:
oh, you can just do this on the PR itself and get proper notifications related to that:
<img width="319" alt="Screen Shot 2020-05-13 at 4 04 30 PM" src="https://user-images.githubusercontent.com/487433/81874765-75c5f600-9533-11ea-9ea3-17a8c255ae14.png">

The person opening the PR probably doesn't know who should review, so having this in the template makes outsiders feel even more that way. 

There are ways to auto-assign based on which folders/modules are touched we could use in the future.

Reply:
It seems only repo collaborators can request reviewers from there. Do all committers get notified when a new PR is created?

Reply:
Oh, we should add a feature to let anyone assign PRs through comments on the apache superset Github bot:
https://github.com/apache-superset/superset-github-bot

It can be a way to work around missing perms for contributors (non-committers). 

The labeling feature broke recently for reasons I could not understand (felt like unicode-related around parsing the emoji). Spent 2 hours and gave up.